vrtool_docs/Preprocessing/Bekleding.md

```diff
@@ -4,22 +4,20 @@
 
 De basis voor het genereren van de bekleding berekeningen is het invoerbestand `Bekleding_default.csv`. Dit bestand is terug te vinden in: ```.\VRSuiteUtils-main\preprocessing\default_files``` in de ZIP bestand die bij de installatie voor [preprocessing](..\Installaties\VRUtils.md) is gedownload.
 
-Dit bestand heeft de volgende kolommen die ingevuld moeten worden:
+Dit bestand heeft de volgende kolommen die ingevul dmoeten worden:
 
 | Kolom       	          | 	           | Beschrijving                                                                                                                                                                                 	 |
 |------------------------|-------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| doorsnede 	            | Verplicht 	 | De vaknaam zoals beschreven in de Vakindeling.csv                                                                                                                                              |
-| naam_hr_locatie    	 | Verplicht 	 | Naam van de hr_locatie (doorsnede) zoals gebruikt bij de voorbereidende waterstandsberekeningen                                                                                             |
```

Comment:
Deze moet er wel bij blijven! In default excel heet het 'HR_locatie'

Reply:
Nee dat is inmiddels aangepast